### ISSUE TITLE
Walking out of streaming range with attached object breaks streamer

### ISSUE THREAD

**Issue by datWeazel** [gurkansahinn/object-streamer#2]
Sorry for the weird title.

How to reproduce: 
- Create a new attached object (attached to yourself)
- Walk out of streaming range
- Object should now vanish from the body

The underlying issue is that the ObjectStreamer.position is never updated. So if a player "carries" an item too far from the original spawn position, the object counts as no longer in range even tho it's at the players body.

**Comment by gurkansahinn** [comment]
Yes, constantly updating this can cause problems at some point. It can be highlighted after Octree implementation. Thanks for this issue. @datWeazel
